cilium/cmd/cleanup.go

```diff
@@ -511,3 +533,47 @@ func removeTCFilters(linkAndFilters map[string][]*netlink.BpfFilter) error {
 
 	return nil
 }
+
+func removeXDPs(links []netlink.Link) error {
+	for _, link := range links {
+		err := netlink.LinkSetXdpFd(link, -1)
+		if err != nil {
+			return err
+		}
+		log.Infof("removed cilium xdp of %s", link.Attrs().Name)
```

Comment:
I don't see other functions using `log.Info` but `fmt.Printf` instead. Which should we be using?